src/styles/item/_index.scss

```diff
@@ -334,8 +330,9 @@
                 color: var(--text-light);
                 display: flex;
                 justify-content: space-between;
-                margin: -1px -1px 0 -1px;
-                padding: 1px;
+                margin: var(--space-neg-1) var(--space-neg-1) 0 var(--space-neg-1);
+                padding: var(--space-1);
```

Comment:
```suggestion
                margin: var(--space-neg-1) var(--space-neg-1) 0 var(--space-neg-1);
                padding: var(--space-1);
```

These are in use in affliction to do the stage header. Test there to make sure nothing broke, but I think these properties exist in order to cause it to overlap with the border of the content.
![image](https://github.com/foundryvtt/pf2e/assets/1286721/7b5599c1-7946-4a4b-a936-144614b1949b)